src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -345,6 +347,54 @@ private function registerProfilerConfiguration(array $config, ContainerBuilder $
     }
 
     /**
+     * Loads the workflow configuration.
+     *
+     * @param array            $workflows A workflow configuration array
+     * @param ContainerBuilder $container A ContainerBuilder instance
+     * @param XmlFileLoader    $loader    An XmlFileLoader instance
+     */
+    private function registerWorkflowConfiguration(array $workflows, ContainerBuilder $container, XmlFileLoader $loader)
+    {
+        if (!$workflows) {
+            return;
+        }
+
+        $loader->load('workflow.xml');
+
+        $registryDefintion = $container->getDefinition('workflow.registry');
+
+        foreach ($workflows as $name => $workflow) {
+            $definitionDefinition = new Definition('Symfony\Component\Workflow\Definition');
+            $definitionDefinition->addMethodCall('addPlaces', [$workflow['places']]);
```

Comment:
I know. This part is not updated yet. But anyway I'm going to revert the last change about the Place object.